src/overlays/actors/ovl_Obj_Um/z_obj_um.c

```diff
@@ -46,131 +73,1951 @@ const ActorInit Obj_Um_InitVars = {
+    bandit->banditPosition.z =
+        bandit->actor.world.pos.z + (bandit->actor.velocity.z * 0.5f) + bandit->actor.colChkInfo.displacement.z;
+
+    phi_v1_2 = BINANG_SUB(bandit->actor.world.rot.y, bandit->actor.shape.rot.y);
+
+    if (phi_v1_2 > 0x190) {
+        phi_v1_2 = 0x190;
+    } else if (phi_v1_2 < -0x190) {
+        phi_v1_2 = -0x190;
+    }
+
+    bandit->actor.shape.rot.y = bandit->actor.shape.rot.y + phi_v1_2;
+    return 0;
+}
+
+// ObjUm_Bandit_GetCloserAndAttack
+s32 func_80B78764(ObjUm* this, PlayState* play, EnHorse* bandit1, EnHorse* bandit2) {
+    s32 pad;
+    Vec3f sp30;
+    s16 phi_v1_5;
+
+    bandit1->unk_55C--;
+    if (bandit1->unk_55C <= 0) {
+        bandit1->unk_55C = 0;
+
+        if ((bandit1->unk_550 == 3) && !(this->flags & OBJ_UM_FLAG_MINIGAME_FINISHED)) {
+            s32 potIndex = -1;
+
+            if (this->potsLife[0] != 1) {
+                potIndex = 0;
+            }
+
+            if (this->potsLife[1] != 1) {
+                if ((potIndex == -1) || (potIndex == 0 && Rand_ZeroOne() < 0.3f)) {
+                    potIndex = 1;
+                }
+            }
+
+            if (this->potsLife[2] != 1) {
+                if ((potIndex == -1) || (potIndex != -1 && Rand_ZeroOne() < 0.3f)) {
+                    potIndex = 2;
+                }
+            }
+
+            if (this->potsLife[potIndex] != 1) {
+                this->wasPotHit[potIndex] = true;
+                if (this->potsLife[potIndex] == 2) {
+                    Audio_PlaySfxAtPos(&this->potPos[potIndex], NA_SE_EV_MILK_POT_BROKEN);
+                } else {
+                    Audio_PlaySfxAtPos(&this->potPos[potIndex], NA_SE_EV_MILK_POT_DAMAGE);
+                }
+
+                this->potsLife[potIndex]--;
+                if (this->potsLife[potIndex] < 1) {
+                    this->potsLife[potIndex] = 1;
+                }
+            }
+        }
+
+        func_80B781DC(this, bandit1, bandit2, play);
+    }
+
+    Math3D_Lerp(&bandit1->unk_540, &this->unk_360[bandit1->unk_550], 1.0f - ((f32)bandit1->unk_55C / bandit1->unk_560),
+                &sp30);
+    bandit1->banditPosition = sp30;
+    bandit1->unk_588 = this->dyna.actor.shape.rot.y;
+
+    if ((bandit1->unk_550 == 10) || ((bandit1->unk_550 == 8))) {
+        phi_v1_5 = bandit1->unk_588;
+    } else if (Math3D_Distance(&bandit1->actor.prevPos, &bandit1->actor.world.pos) < 10.0f) {
+        phi_v1_5 = bandit1->unk_588;
+    } else {
+        phi_v1_5 = Math_Vec3f_Yaw(&bandit1->actor.prevPos, &bandit1->actor.world.pos);
+    }
+
+    if (1) {}
+
+    phi_v1_5 -= bandit1->actor.shape.rot.y;
+    if (phi_v1_5 > 0x190) {
+        bandit1->actor.shape.rot.y += 0x190;
+    } else if (phi_v1_5 < -0x190) {
+        bandit1->actor.shape.rot.y -= 0x190;
+    } else {
+        bandit1->actor.shape.rot.y += phi_v1_5;
+    }
+
+    return 0;
+}
+
+// ObjUm_UpdateBandits...something
```

Comment:
This looks like something checking to see if they're damaged. We normally call these something like "UpdateDamage", but I understand if you don't want to actually name the function. At least this comment can be updated though.